package.json

```diff
@@ -11,6 +11,6 @@
   },
   "homepage": "https://github.com/18F/identity-partners-site#readme",
   "dependencies": {
-    "identity-style-guide": "^2.2.1"
+    "identity-style-guide": "^2.2.2"
```

Comment:
I just tried this locally... I think you need to run `npm upgrade --save identity-style-guide` so that it updates the `yarnfile.lock` too, otherwise they get sad when they get out of sync